android/app/src/main/kotlin/coop/polypoly/polypod/MainActivity.kt

```diff
@@ -24,16 +27,21 @@ class MainActivity : AppCompatActivity() {
         val notification = UpdateNotification(this)
         notification.handleStartup()
 
-        if (Preferences.isFirstRun(this)) {
+        val firstRun = Preferences.isFirstRun(this)
+        if (firstRun) {
             notification.handleFirstRun()
+        }
+
+        val shouldShowOnboarding =
+            firstRun || Authentication.shouldShowBiometricsPrompt(this)
+        if (!onboardingShown && shouldShowOnboarding) {
```

Comment:
This is the new logic for showing the onboarding: Previously the onboarding would basically _close itself_ when it noticed that it's neither the first run, nor should the user be reminded to set up biometrics.